digitalocean/resource_digitalocean_container_registry_test.go

```diff
@@ -0,0 +1,94 @@
+package digitalocean
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/digitalocean/godo"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccDigitalOceanContainerRegistry_Basic(t *testing.T) {
+	var reg godo.Registry
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckDigitalOceanContainerRegistryDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckDigitalOceanContainerRegistryConfig_basic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDigitalOceanContainerRegistryExists("digitalocean_container_registry.foobar", &reg),
+					testAccCheckDigitalOceanContainerRegistryAttributes(&reg),
+					resource.TestCheckResourceAttr(
+						"digitalocean_container_registry.foobar", "name", "foobar"),
+				),
```

Comment:
```suggestion
					resource.TestCheckResourceAttr(
						"digitalocean_container_registry.foobar", "write", "false"),
					resource.TestCheckResourceAttrSet(
						"digitalocean_container_registry.foobar", "docker_credentials"),
					resource.TestCheckResourceAttr(
						"digitalocean_container_registry.foobar", "endpoint", "registry.digitalocean.com/foobar"),
				),
			},
			{
				Config: testAccCheckDigitalOceanContainerRegistryConfig_write,
				Check: resource.ComposeTestCheckFunc(
					testAccCheckDigitalOceanContainerRegistryExists("digitalocean_container_registry.foobar", &reg),
					testAccCheckDigitalOceanContainerRegistryAttributes(&reg),
					resource.TestCheckResourceAttr(
						"digitalocean_container_registry.foobar", "name", "foobar"),
					resource.TestCheckResourceAttr(
						"digitalocean_container_registry.foobar", "write", "true"),
				),
			},
```

It would be great to test the update path as well as make sure other attributes are being set.